cdc/capture/capture.go

```diff
@@ -131,6 +131,16 @@ func NewCaptureWithManager4Test(o owner.Owner, m *upstream.Manager) *Capture {
 	return res
 }
 
+// GetUpstreamManager is a Getter of capture's upstream manager
+func (c *Capture) GetUpstreamManager() *upstream.Manager {
+	return c.UpstreamManager
```

Comment:
Can we change the field `UpstreamManager` to `upstreamManager` to prevent external packages access it directly?